serializers/avro/src/main/java/io/pravega/schemaregistry/serializer/avro/impl/AvroDeserializer.java

```diff
@@ -39,7 +42,7 @@
     }
 
     @Override
-    protected T deserialize(InputStream inputStream, SchemaInfo writerSchemaInfo, SchemaInfo readerSchemaInfo) throws IOException {
+    public final T deserialize(InputStream inputStream, SchemaInfo writerSchemaInfo, SchemaInfo readerSchemaInfo) throws IOException {
```

Comment:
What is the reason for preventing overrides?

Reply:
these are final implementation of a specific typed serializer. 
We dont need them to be overridable. hence made them public final here and in all the final implementation classes.